Source/Core/Core/HW/CPU.cpp

```diff
@@ -10,12 +10,15 @@
 #include "AudioCommon/AudioCommon.h"
 #include "Common/CommonTypes.h"
 #include "Common/Event.h"
+#include "Common/Timer.h"
+#include "Core/ConfigManager.h"
 #include "Core/Core.h"
 #include "Core/Host.h"
 #include "Core/PowerPC/GDBStub.h"
 #include "Core/PowerPC/PowerPC.h"
 #include "Core/System.h"
 #include "VideoCommon/Fifo.h"
+#include <Core/TimePlayed.h>
```

Comment:
We use regular quotes for non-system includes; plus it needs to be sorted alphabetically (up by one line)
```suggestion
#include "Core/TimePlayed.h"
```